### PR TITLE
Node: make connection metrics optional

### PR DESCRIPTION
**Pull request by Raynos**
This PR allows you to turn connection metrics off. Specifically
the bytes recvd/sent and the gauge are expensive.

r: @kriskowal @shannili @rf

### CODE REVIEW

File: node/v2/handler.js [hunk 1]
```diff
@@ -212,16 +212,25 @@ TChannelV2Handler.prototype.handleCallRequest = function handleCallRequest(reqFr
...
-        new ConnectionsBytesRcvdTags(
-            channel.hostPort || '0.0.0.0:0',
-            self.connection.socketRemoteAddr
-        )
-    ));
+    self.emitBytesRecvd(reqFrame);
+};
 
+TChannelV2Handler.prototype.emitBytesRecvd =
+function emitBytesRecvd(frame) {
```
Comment (kriskowal) [inline node/v2/handler.js]: Is there precedent for compressing "Received" to "Recvd" (I mean, if we ignore C libraries)?

**Comment by ShanniLi** [comment]
lgtm. Good change! Connection metrics seem not that critical in practice anyway.

**Comment by kriskowal** [comment]
:ok:

File: node/v2/handler.js (hunk 1 above)
Comment (Raynos) [inline node/v2/handler.js]: it's `bytes-recvd` which is the stat name; i don't care what we call this method.

Comment (kriskowal) [inline node/v2/handler.js]: Alright, I’ll shrug.

**Pull request merged**